packages/payment-detection/test/any/any-to-erc20-proxy-contract.test.ts

```diff
@@ -167,7 +209,8 @@ describe('api/any/conversion-fee-proxy-contract', () => {
     });
   });
 
-  it('can get the fees out of payment events, and payment & refund work even with the wrong feeAddress while feeBalance sum only if feeAddress is fine', async () => {
+  // TODO: not sure about this
```

Comment:
what about it? does the test pass? you don't need to modify all tests it's OK to leave some rinkeby tests